### DISCUSSION TITLE
Size increase of the leaderboard + chatbox during a multiplayer game

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#25841] [Ideas]
Don't get me wrong, having the simple design for both of these is nice, and I believe that it should be that way by default, but it would be nice to have a third option (apart from simply hiding and displaying the small size), which is an expanded version which makes the leaderboard profile cards a bit bigger and especially the chatbox being bigger, which is a big issue that I found when trying out multiplayer.

**Comment by marioman63** [comment]
I believe this is how it used to be (showed full names of players and bigger chat, like it does during breaks but throughout the song all the time), but this had the issue of overlapping hit objects during gameplay. I also liked it expanded though and didn't find it that intrusive, so an option for it would be nice. Also since UI customization is more fleshed outnow  it could always become a skinnable element (if it isn't already) so that it doesn't overlap anything.

**Reply by 19peytonsmith** [reply]
Yeah... for me it's mostly the chatbox, even during breaks the chatbox is super small which is really annoying to see messages well when a sentence breaks into 5 lines and takes up the entire view.
